css/style.css

```diff
@@ -0,0 +1,1140 @@
+    color: #E51937;
+    text-shadow: none;
+    margin-left: 5px;
+    font-size: 13px;
+  }
+
+  .main__arrow {
+    margin: 0 auto;
+  }
+}
+
+/* .container {
+  max-height: 900px;
+} */
+
+.container__title {
+  color: #E51937;
+  margin: 101px 905px 0 80px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 70px;
+  line-height: 85px;
+  z-index: 99;
+}
+
+.container__content {
+  display: flex;
+}
+
+.container__img {
+  display: inline-block;
+  width: 100%;
+  margin-top: -100px;
+  z-index: -1;
+}
+
+.container__img-phone {
+  display: none;
+}
+
+.container__aside {
+  display: flex;
+  flex-direction: column;
+  background-color: #E51937;
+  margin-left: -270px;
+  margin-top: -151px;
+  padding: 20px 51px 0 48px;
+}
+
+.title {
+  text-align: left;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 37px;
+  color: #fff;
+}
+
+.title-second {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+}
+
+.container__aside-info {
+  margin-top: -14px;
+  text-align: left;
+  margin-bottom: 27px;
+}
+
+.title-second_white {
+  color: #fff;
+}
+
+.container__boxes {
+  justify-content: space-between;
+}
+
+.container__box {
+  display: flex;
+  margin-bottom: 30px;
+}
+
+.box-info {
+  display: flex;
+  flex-direction: column;
+}
+
+.container__img-screen {
+  margin-top: 38px;
+}
+
+.container__box-title {
+  display: flex;
+  align-items: flex-end;
+  font-size: 35px;
+  font-weight: 600;
+  color: white;
+  margin-top: 31px;
+  margin-left: 23px;
+}
+
+.box-title__subtitle {
+  display: flex;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  margin-left: 10px;
+  margin-top: -5px;
+}
+
+.container__box-desc {
+  display: flex;
+  color: white;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  margin-top: -37px;
+  margin-left: 22px;
+  padding-right: 87px;
+}
+
+.container__box-update {
+  display: flex;
+  padding-right: 10px;
+  line-height: 18px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  color: white;
+  margin: 5px 0 0 17px;
+}
+
+.container__box-functionality {
+  display: flex;
+  margin: 4px 77px 0 15px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+  color: white;
+}
+
+.box-bag {
+  margin-top: -36px;
+}
+
+.container__img-bag {
+  margin-top: -8px;
+}
+
+.box-about {
+  display: flex;
+  align-items: flex-end;
+}
+
+.box-about__container {
+  display: flex;
+  justify-content: space-between;
+  margin: 128px -12px 63px -1px;
+}
+
+.box-about__link {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: white;
+  text-decoration: none;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+}
+
+.box-about__learn {
+  display: flex;
+  flex-wrap: nowrap;
+  margin-left: 9px;
+}
+
+.box-about__button {
+  justify-content: center;
+  width: 160px;
+  margin-left: 33px;
+  border: white 2px solid;
+  color: white;
+  background: #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla-Project_Evgen/pullRequest?prid=6071575)